features/git-town/configuration/parent_branch.feature

```diff
@@ -0,0 +1,34 @@
+Feature: update the parent of a nested feature branch
+
+  As a user with a nested feature branch shipped whose parent was shipped from another machine
+  I want to be able to update the parent branch for my nested feature branch
+  So that I can use it without messing with the git configuration directly
```

Comment:
Let me try and think when I needed this. I think it was mainly with two machines where each knows about parent-feature and child-feature. I ship parent-feature on one machine and then need to update child-feature's parent on another branch. I guess this could be solved with prune-branches updating the parent branches as needed.